content/sensu-core/0.29/guides/troubleshooting-sensu.md

```diff
@@ -0,0 +1,322 @@
+
+{{< highlight shell >}}
+$ ps aux | grep [s]ensu-server
+sensu     5992  1.7  0.3 177232 24352 ...
+$ kill -TRAP 5992{{< /highlight >}}
+
+Additionally, you can set the log level to `info` or `debug` by using the configuration directive in `/etc/default/sensu`. Let's take a look at an example:
+
+{{< highlight shell >}}
+# cat /etc/default/sensu
+LOG_LEVEL=info{{< /highlight >}}
+
+And after setting that directive, restarting the respective Sensu services:
+
+{{< highlight shell >}}
+sudo systemctl restart sensu-{server,api,client}{{< /highlight >}}
+
+Keep in mind that to set log levels back to normal, you can either run `kill TRAP $SENSUPID` (if you've used that method), or revert the change in `/etc/default/sensu` and restart the Sensu processes for the change to take place.
+
+### Printing Configurations
+
+Frequently, Sensu staff or community members may ask you to print your configuration. It's fairly easy to print the configuration for your Sensu deployment:
+
+**Sensu Core**:
+`/opt/sensu/bin/sensu-client --print_config | tee sensu-core-config.json`
+
+**Sensu Enterprise**
+`sudo -u sensu java -jar /usr/lib/sensu-enterprise/sensu-enterprise.jar -c /etc/sensu/config.json -d /etc/sensu/conf.d --print_config | tee se-config.json`
+
+This command will result in output that will list the entire configuration for your Sensu deployment. This can be especially useful when comparing the configuration that Sensu is aware of, versus the configuration living on-disk. If the values of a particular file differ from what you're expecting, then see the next section for how to proceed.
+
+### Restarting Services
+
+It's crucial that you restart your Sensu services after each change so that the configuration changes are read. For most modern Linux distributions, this is done using `systemd`:
+
+{{< highlight shell >}}
+sudo systemctl restart sensu-{server,api,client}{{< /highlight >}}
+
+In the event that you're using a system where `sysvinit` is the service manager of choice, you can use:
+
+{{< highlight shell >}}sudo service sensu-client restart
+sudo service sensu-server restart
+sudo service sensu-api restart{{< /highlight >}}
+
+It's especially important to restart the `sensu-client` process if you're making use of any [standalone][] checks, as the client will be responsible for check scheduling and execution.
+
+### Collecting Logs
+
+Sensu's logs provide a wealth of information when troubleshooting issues. They live at `/var/log/sensu`:
+
+{{< highlight shell >}}# tree /var/log/sensu
+/var/log/sensu
+├── sensu-api.log
+├── sensu-client.log
+└── sensu-server.log{{< /highlight >}}
+
+Sensu staff, or community members may ask to see your logs. You can view them at the paths above, or provide them in an archive:
+
+{{< highlight shell >}}
+tail -n 10000 /var/log/sensu/sensu-server.log > sensu-server-10k.log && gzip -9 sensu-server-10k.log`{{< /highlight >}}
```

Comment:
` at the end of the command would cause the command to not run properly.

Reply:
✔️